src/boxes/media/mod.rs

```diff
@@ -73,27 +76,18 @@ impl Mp4Box for Media {
 
             match name {
                 AtomName::MediaInfo => {
-                    let b = Box::new(MediaInfo::parse(
-                        reader,
-                        index,
-                        level + 1,
-                    )?) as Box<dyn Mp4Box>;
+                    let b =
```

Comment:
Have you used `cargo fmt` to auto-format the code? There are many formatting-related changes.

Reply:
Yes, that is correct. I think this can be ignored as this is not making changes to the logic of the code?